source/Octopus.Tentacle.Tests.Integration/Support/IntegrationTest.cs

```diff
@@ -1,5 +1,4 @@
 ﻿using System;
-using System.Diagnostics;
 using System.Threading;
```

Comment:
Simplification of this class to align with the implementation in Halibut and remove some possible issues our test infra creates